### PR TITLE
Pattern: Use one line for each declaration, it enhances code readabil…

### PR DESCRIPTION
**Pull request by tsvetaevAlex**
29 files committed: Pattern: Use one line for each declaration, it enhances code readability. 

Part 1. file list 3 of 4

### CODE REVIEW

File: jdi-light-bootstrap-tests/src/main/java/io/github/com/entities/User.java [hunk 1]
```diff
@@ -4,5 +4,12 @@
 import com.epam.jdi.tools.DataClass;
 
 public class User extends DataClass<User> {
-    public String name = "Roman", password = "Jdi1234";
+    public String name = "Roman";
+    public String
+
```
Comment (evasileva-epam) [inline jdi-light-bootstrap-tests/src/main/java/io/github/com/entities/User.java:9]: Please delete extra lines

File: jdi-light-html-tests/src/main/java/io/github/com/entities/MarvelUserInfo.java [hunk 2]
```diff
@@ -3,5 +3,9 @@
 import com.epam.jdi.tools.DataClass;
 
 public class MarvelUserInfo extends DataClass<MarvelUserInfo> {
-    public String number, type, user, description;
+    public String;
```
Comment (evasileva-epam) [inline jdi-light-html-tests/src/main/java/io/github/com/entities/MarvelUserInfo.java:6]: Line 6 is not required

File: jdi-light-html-tests/src/main/java/io/github/com/pages/HtmlElementsPage.java [hunk 3]
```diff
@@ -97,7 +105,8 @@
     public static MultiSelector ages;
     //@UI("#ages") public static MultiSelector ages;
     public static MultiSelector multiDropdown;
-    // @UI("#multi-dropdown") public static MultiSelector multiDropdown;
+    // @UI("#multi-dropdown") public static
+    // public static multiDropdown;
```
Comment (evasileva-epam) [inline jdi-light-html-tests/src/main/java/io/github/com/pages/HtmlElementsPage.java:109]: Please return to previous variant on one line

File: jdi-light/src/main/java/com/epam/jdi/light/driver/sauce/SauceSettings.java [hunk 4]
```diff
@@ -9,14 +9,18 @@
...
-        accessKey = getenv("SAUCE_ACCESS_KEY"),
-        seleniumVersion, name = "Jdi tests",
-        maxDuration = "3600",
-        commandTimeout = "600",
-        idleTimeout = "1000",
-        build = "JDI uild",
-        browserName, browserVersion, platformName, screenResolution, timeZone;
+    public static String username = getenv("SAUCE_USERNAME");
+    public static String accessKey = getenv("SAUCE_ACCESS_KEY");
+    public static String seleniumVersion, name = "Jdi tests";
```
Comment (evasileva-epam) [inline jdi-light/src/main/java/com/epam/jdi/light/driver/sauce/SauceSettings.java:14]: seleniumVersion and name should be on different lines

**Review by evasileva-epam — CHANGES_REQUESTED** [review]

File: jdi-light-bootstrap-tests/src/main/java/io/github/com/entities/User.java (hunk 1 above)
Comment (tsvetaevAlex) [reply jdi-light-bootstrap-tests/src/main/java/io/github/com/entities/User.java]: exrta empty lines remiver

File: jdi-light-html-tests/src/main/java/io/github/com/entities/MarvelUserInfo.java (hunk 2 above)
Comment (tsvetaevAlex) [reply jdi-light-html-tests/src/main/java/io/github/com/entities/MarvelUserInfo.java]: removed

File: jdi-light/src/main/java/com/epam/jdi/light/driver/sauce/SauceSettings.java (hunk 4 above)
Comment (evasileva-epam) [inline jdi-light/src/main/java/com/epam/jdi/light/driver/sauce/SauceSettings.java:14]: Not fixed

**Review by evasileva-epam — CHANGES_REQUESTED** [review]

**Review by tsvetaevAlex — COMMENTED** [review]
fixed. separated to different lines

File: jdi-light-html-tests/src/main/java/io/github/com/pages/HtmlElementsPage.java (hunk 3 above)
Comment (tsvetaevAlex) [reply jdi-light-html-tests/src/main/java/io/github/com/pages/HtmlElementsPage.java:109]: recovered to:
"// @UI("#multi-dropdown") public static multiDropdown;"

**Pull request merged**